app/agent/collector.scala

```diff
@@ -32,23 +34,27 @@ class CollectorAgent[T<:IndexedItem](val collectorSet: CollectorSet[T], sourceSt
 
   def size: Int = get().map(_.data.size).sum
 
+  override def toMarkerMap: Map[String, Any] = Map("records" -> size)
+
   def update(collector: Collector[T], previous:Datum[T]):Datum[T] = {
     val s = new StopWatch
     val datum = Datum[T](collector)
     val timeSpent = s.elapsed
     sourceStatusAgent.update(datum.label)
     datum.label match {
       case l@Label(product, origin, size, _, None) =>
-        log.info(s"Crawl of ${product.name} from $origin successful (${timeSpent}ms): $size records, ${l.bestBefore}")
+        val marker = Markers.appendEntries((origin.toMarkerMap ++ l.toMarkerMap ++ this.toMarkerMap ++ Map("duration" -> timeSpent)).asJava)
```

Comment:
Just to make review easier, I've listed the fields here too:
1. `origin.toMarkerMap` holds our **region** field
2. `l.toMarkerMap` holds our **resource** and **accounts** fields
3. `this.toMarkerMap` is the **records** field on line 37 of this class 
4. and then finally we include the **duration** field here in the `update` function where we have access to the stopwatch